createdisk-library.sh

```diff
@@ -170,6 +170,45 @@ function copy_additional_files {
     eventually_add_pull_secret $destDir
 }
 
+function prepare_hyperV() {
+
+    if [ "${ARCH}" != "ppc64le" ]; then
+        echo "hyperv is not available on ppc64le"
+        return
+    fi
+
+    if [[ ${OKD_VERSION} != "none" ]]; then
+        # Install the hyperV and libvarlink-util rpms to VM
+        ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora.repo'
+        ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora-updates.repo'
+        ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install --allow-inactive hyperv-daemons libvarlink-util'
+        ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo sed -i -z s/enabled=1/enabled=0/ /etc/yum.repos.d/fedora.repo'
+        ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo sed -i -z s/enabled=1/enabled=0/ /etc/yum.repos.d/fedora-updates.repo'
+    else
+        # Download the hyperV daemons and libvarlink-util dependency on host
+        mkdir $1/packages
+        sudo yum install -y --downloadonly --downloaddir $1/packages hyperv-daemons libvarlink-util
```

Comment:
This is reverting the changes from https://github.com/code-ready/snc/commit/24f2971a964d9961bdb7fc646d56bb93419bdb79